script.xbmcbackup/resources/data/default_files.json

```diff
@@ -0,0 +1,105 @@
+{
```

Comment:
`__init__py` files should not be needed with python

Reply:
Didn't know these could be removed in Python 3 - thanks!